### PR TITLE
make submission requirements clearer

### PR DESCRIPTION
**Pull request by mmulholla**
Added info to submission requirements to address: https://issues.redhat.com/browse/HELM-223

see also: https://github.com/openshift-helm-charts/development/pull/93

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -66,11 +66,20 @@ You can submit chart related changes in three methods:
 1. Submit a chart without Chart Verification Report
 
    - Chart as a tarball
+      - must be created using ```helm package```
```
Comment (zonggen) [inline docs/README.md:69]: I think this should be "**can** be created using `helm package`". A user can also do `tar -cvzf <target>.tgz <chart-directory>`.

Comment (tisutisu) [reply docs/README.md:69]: I think 'helm package' command should be recommended as by default the resulting tar file is of format <chart-name>-<chart-version>.tgz .While for tar command user have to specify the target tgz file name, which is prone to user error.

Comment (mmulholla) [reply docs/README.md]: @zonggen @tisutisu have update based on your suggestions.

**Review by zonggen — APPROVED** [review]

**Review by tisutisu — APPROVED** [review]
LGTM

**Pull request merged**